cpp/box/export-box.h

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2010-2023 The Regents of the University of Michigan
+// This file is from the freud project, released under the BSD 3-Clause License.
+
+#ifndef EXPORT_BOX_H
+#define EXPORT_BOX_H
+
+#include "Box.h"
+
+#include <nanobind/ndarray.h>
+namespace nb = nanobind;
```

Comment:
This code is intended to set the pattern for the rest of freud. Other modules will be significantly bigger and I thought it might be cumbersome to have so much wrapper code in the same file. Additionally, we may want to separate this translation unit into an `export-box.h` and an `export-box.cc` (or multiple translation units) to improve compile times with larger modules.